### PR TITLE
net-libs/gssdp,  net-libs/gupnp: Version bump to 1.6.2 

### PR DESCRIPTION
**Pull request by flint2**

### CODE REVIEW

File: net-libs/gssdp/gssdp-1.6.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+"
+
+# has file collision with Slot 0 (/usr/bin/gssdp-device-sniffer), so we remove that from this build
+# and depend on the other slot that has the file as well
+RDEPEND="
+	>=dev-libs/glib-2.70:2[${MULTILIB_USEDEP}]
+	>=net-libs/libsoup-2.99.0:3.0[${MULTILIB_USEDEP},introspection?]
+	gtk? ( gui-libs/gtk:4 )
+	introspection? ( >=dev-libs/gobject-introspection-1.54:= )
+	manpages? ( app-text/pandoc )
```
Comment (mattst88) [inline net-libs/gssdp/gssdp-1.6.2.ebuild:27]: I think this should be `virtual/pandoc` instead.

File: net-libs/gssdp/gssdp-1.6.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+inherit gnome.org meson-multilib vala xdg
+
+DESCRIPTION="GObject-based API for handling resource discovery and announcement over SSDP"
+HOMEPAGE="https://wiki.gnome.org/Projects/GUPnP https://gitlab.gnome.org/GNOME/gssdp"
+
+LICENSE="LGPL-2+"
+SLOT="1.6/1.6-0" # <API version>-<soname>
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~loong ~ppc ~ppc64 ~riscv ~sparc ~x86"
+IUSE="gtk gtk-doc +introspection +vala manpages"
```
Comment (mattst88) [inline net-libs/gssdp/gssdp-1.6.2.ebuild:14]: We just use `man` as the USE flag for this. Also, IUSE should be in alphabetical order.

File: net-libs/gssdp/gssdp-1.6.2.ebuild (hunk 1 above)
Comment (mattst88) [reply net-libs/gssdp/gssdp-1.6.2.ebuild:27]: Also, pandoc is not stable, so in preparation for eventually stabilizing gssdp we should add an entry to `base/package.use.stable.mask` masking this new man page flag except on unstable.

**Review by mattst88 — CHANGES_REQUESTED** [review]

File: net-libs/gssdp/gssdp-1.6.2.ebuild (hunk 1 above)
Comment (flint2) [reply net-libs/gssdp/gssdp-1.6.2.ebuild]: Solved in https://github.com/gentoo/gentoo/commit/d9387b41347e672eff6777cbff5ce078e0e58619

File: net-libs/gssdp/gssdp-1.6.2.ebuild (hunk 2 above)
Comment (flint2) [reply net-libs/gssdp/gssdp-1.6.2.ebuild]: Solved in https://github.com/gentoo/gentoo/commit/d9387b41347e672eff6777cbff5ce078e0e58619

**Comment by mattst88** [comment]
Thanks!

I added some additional `package.use.mask` entries so we could keep `KEYWORDS` unmodified. Since we're almost certainly never going to keyword pandoc on most architectures, it makes sense to just mask it.

**Pull request closed without merging**